createDerby.groovy

```diff
@@ -184,6 +185,8 @@ if (hmdbFile.exists()) {
          addXRef(database, ref, rootNode.cas_registry_number.toString(), casDS, genesDone, linksDone);
          addXRef(database, ref, rootNode.pubchem_compound_id.toString(), pubchemDS, genesDone, linksDone);
          addXRef(database, ref, rootNode.chemspider_id.toString(), chemspiderDS, genesDone, linksDone);
+	//   addXRef(database, ref, rootNode.iuphar_id, iupharDS, 
```

Comment:
I think this section is where it added stuff from the HMDB database. I don't think we want that...

Reply:
Okay, makes sense now. I'll remove it.